roles/generate_manifests/README.md

```diff
@@ -1,3 +1,41 @@
-# generate_manifests
+# generate_manifests role
 
-Generates the manifests required for openshift_installer's agent sub-command
\ No newline at end of file
+Generates the manifests required for openshift_installer's agent sub-command
+
+## Variables
+
+All of the variables have sane defaults which you can override to force things, but you shouldn't need to
+
+| Variable                      | Default                           | Required  | Description           |
+| ----------------------------- | --------------------------------- | --------- | --------------------- |
+| generated_dir                 | $REPO_ROOT_PATH/generated         | No        | Base path for all generated files |
+| manifests_dir                 | $GENERATED_DIR/$CLUSTER_NAME      | No        | Path to store all rendered manifests |
+| cluster_manifest_dir          | $MANIFESTS_DIR/cluster-manifests  | No        | Path for cluster manifests |
+| extra_manifest_dir            | $MANIFESTS_DIR/openshift          | No        | Path for extra manifests |
+| static_network_config         | {}                                | No        | Static network config for every node |
+| installation_disk_path        |                                   | No        | Disk to use for install if you don't want the first found disk |
+| root_device_hints             | {}                                | No        | Install device hints per node, in case installation_disk_path is not enough |
```

Comment:
would it be useful to link to documentation describing what hints are available?